src/hotspot/share/gc/parallel/parallelScavengeHeap.hpp

```diff
@@ -74,6 +74,10 @@ class ParallelScavengeHeap : public CollectedHeap {
 
   WorkGang _workers;
 
+  size_t _young_live;
+  size_t _eden_live;
+  size_t _old_live;
```

Comment:
It's only the sum that's ever exposed, right? I wonder if it makes sense to merge them into one var to only track the sum.

Reply:
I agree because they seem to be always read and written at the same time.

Reply:
The original idea was that the sum might be computed from different areas depending on the GC phase. But, apparently, that's not the case so we can have just one common sum.

:+1: